tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -3930,3 +3930,242 @@ Feature: check activities
         }
       }
       """
+
+  @issue-9536
+  Scenario: check activities after moving a file inside a subfolder in a project space
```

Comment:
do we have tests for personal space?

Reply:
Not yet for this same journey for Personal space. Should I write tests for Personal space too?

Reply:
I think following scenario covers this for personal space
https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839

Reply:
> I think following scenario covers this for personal space
> 
> https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839

if so then can we combine this two scenarion in one by using Scenario Outline approach @anon-pradip

Reply:
> > I think following scenario covers this for personal space
> > https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839
> 
> if so then can we combine this two scenarion in one by using Scenario Outline approach @anon-pradip

Doing so, I think, we will not meet the exact journey mentioned in the issue

Reply:
this scenario is for project space. looks like we already have test scenario for personal space

Reply:
we should keep this scenario.